src/designer/elsa-workflows-studio/src/components/editors/properties/elsa-script-property/elsa-script-property.tsx

```diff
@@ -44,8 +47,49 @@ export class ElsaScriptProperty {
     }
   }
 
-  onMonacoValueChanged(e: MonacoValueChangedArgs) {
+  onConvertToJsonSchemaClick(e: Event) {
+    e.preventDefault();
+    window.open('https://extendsclass.com/json-schema-validator.html');
+  }
+
+  onMonacoValueChanged(e: MonacoValueChangedArgs) {    
     this.currentValue = e.value;
+    this.validate(e.value);
+  }
+
+  validate(value: string) {
+    const propertyDescriptor = this.propertyDescriptor;
+    const propertyName = propertyDescriptor.name;
+    const fieldName = propertyName;
+    
+    if (fieldName === "Schema")
+    {
+      this.isJsonSchemaValid = true;
+      if (value == '') return;
+
+      const ajv = new Ajv();
+      let json: object;
+      try{
+        json = JSON.parse(value)
+      }
+      catch (e){
+        this.isJsonSchemaValid = false;
+      }
+
+      if (json != undefined)
+      {
+        try {
+          const validate = ajv.compile(json);
+          const errors = validate.errors;
+          if (errors != null)
+            this.isJsonSchemaValid = false;
+        }
+        catch (e){
+          const err = e;
+          this.isJsonSchemaValid = false;
+        }        
+      }
+    }
```

Comment:
This code should be part of the HttpEndpointPlugin class, since it's the only activity that has the "Schema" property.